pkg/metrics/common.go

```diff
@@ -0,0 +1,23 @@
+package mpcmetrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	MetricsSubsystem = "multi_platform_controller"
+	MetricsNamespace = "redhat_appstudio"
```

Comment:
Any reason why not using new org name `konflux_ci` instead of  `redhat_appstudio` ?

Reply:
That is seems to be still default namespace, as far as I can see from other components.
I'll check with `o11y` guys it that really makes some difference.

Reply:
@mshaposhnik lets get this conversation resolved and then we can merge this

Reply:
After looking at o11y metrics  rules, it seems that namespace is not necessary, i will leave only namespace.
Aviablity metric will have value `multi_platform_controller_available` which is imo self-descriptive enough.